src/components/Sidebar/Sidebar.tsx

```diff
@@ -80,19 +96,32 @@ const Sidebar: React.FC = () => {
       <CssBaseline />
       {/* Upper header */}
       <AppBar position="fixed" className="app-bar">
-        <Toolbar>
-          <IconButton
-            color="inherit"
-            aria-label="open drawer"
-            edge="start"
-            onClick={handleDrawerToggle}
-            className="menu-button"
-          >
-            <MenuIcon />
-          </IconButton>
-          <Typography variant="h6" noWrap>
-            <FormattedMessage {...title} />
-          </Typography>
+        <Toolbar className="toolbar">
+          <Box display="flex" alignItems="center">
+            <IconButton
+              color="inherit"
+              aria-label="open drawer"
```

Comment:
Maybe, we need move it to intl if it displayable text

Reply:
It isn't displayable so we can miss its translation